osm2gtfs/creators/incofer/trips_creator_incofer.py

```diff
@@ -79,8 +79,8 @@ def _create_service_period(self, feed, operation):
             if service is not None:
                 return service
         except KeyError:
-            print("INFO. No existe el service_period para la operación:",
-                  operation, " por lo que será creado")
+            print("INFO. No existe el service_period para la operacion:",
+                  operation, " por lo que sera creado")
```

Comment:
¿Por qué quitaste las tildes aquí?

Reply:
To make it work :smile: because of annoying encoding errors. Anyway it probably should be English?!

Reply:
I will changed it to English after this PR get merged.